src/main/java/pacman/ActionKeyPacman.java

```diff
@@ -54,4 +62,20 @@ public void keyReleased(final KeyEvent event) {
      */
     public void keyTyped(final KeyEvent event) {
     }
+
+    /**
+     * Method to set variable assigned to keyRight, keyLeft, keyUp, keyDown
+     */
+    public String assignMovement(final int keyValue) {
+        if (keyValue == KeyEvent.VK_RIGHT) {
+            posToMove = "right";
+        } else if (keyValue == KeyEvent.VK_LEFT) {
+            posToMove = "left";
+        } else if (keyValue == KeyEvent.VK_DOWN) {
+            posToMove = "down";
+        } else if (keyValue == KeyEvent.VK_UP) {
+            posToMove = "up";
```

Comment:
podrias, usar las constantes que tienes declaradas y poner  por ejemplo   posToMove = MOVE_RIGHT y asi con los otros

Reply:
thank you, it's fixed in the last commit